File: api/code-samples/python/klaytnDebug/profiling/writeMemProfile.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

file = "mem.profile"

sdk = OpenSDK(host)
debug_response = sdk.debug.write_mem_profile(file)

print(json.loads(debug_response.response.data))

File: web3rpc/rpc-specs/code-samples/javascript/debug/profilling/writeMemProfile.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const file = "mem.profile";

  provider.debug
    .writeMemProfile(file, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
